picoCTF-web/api/db.py

```diff
@@ -53,6 +55,12 @@ def index_mongo():
     log.debug("Ensuring mongo is indexed.")
 
     db.exceptions.create_index([('time', pymongo.DESCENDING)])
+    db.users.create_index("uid", unique=True, name="unique uid")
+    db.users.create_index("username", unique=True, name="unique usernames")
+    db.users.create_index("username", unique=True, collation=Collation(
+            locale="en", strength=CollationStrength.PRIMARY
+        ), name="unique normalized usernames")
+    db.users.create_index("tid")
```

Comment:
I think some of this is duped with the users indexes below. I also added indexes for the demo.parentemail, email in consideration of the management panel user search, but that kinda goes to question if we really need such a thing for one-off manual queries like that

Reply:
Or rather, should be considered when this step 1: INDEX ALL THE THINGS moves to step 2: trim unnecessary indexes